### ISSUE TITLE
Favorites

### ISSUE THREAD

**Issue by FadiChaalab** [KarimElghamry/chillify#27]
Hi am new in flutter.I try it your way to save favorite songs but am not able to find saved songs after restarting the app, i need help

**Comment by KarimElghamry** [comment]
Hello Fadi, thank you for checking out Chillify!

I am using the `shared_preferences` package to save data onto the device. Basically, there are three major steps to perform:


- Convert the list of favorites to a list of strings

- Save the list using shared preferences

- Load the List on startup and change it back to song objects, then push it to the favorites stream


If you need further assistance, do not hesitate to comment and I will help you further.

**Comment by FadiChaalab** [comment]
Yeah i know that shared preferences is the way to save data but it seems that am having problem with retrieving it after restarting the app it will show as an empty screen

**Comment by ashishkhuraishy** [comment]
@KarimElghamry i think it would be better to use local storage instead of `shared_preferences`, they are commonly used to store a single variable as user name and such.

**Comment by huzaifansari54** [comment]
hello sir can i work on this issue i  want to contribute on this project kindly help me out
